src/Humanizer.Tests.Shared/DateHumanizeDefaultStrategyTests.cs

```diff
@@ -88,8 +88,8 @@ public void HoursFromNowNotTomorrow(int hours, string expected)
 
         [Theory]
         [InlineData(1, "yesterday")]
-        [InlineData(10, "10 days ago")]
-        [InlineData(27, "27 days ago")]
```

Comment:
given this test passes in TimeUnit.Day, why would it result in week/weeks

also 10 days is no "on week"

Reply:
Good point, I think this is going to be a culture thing and then a question of how precise people would like humanizer to be.

7 days would be "a week ago", and in fact if someone asked me about something 8 days ago I might also accept "a week ago".

Perhaps I should look at making this only provide weeks if the value is one of 7,14,21,28. And then fall backs to months as expected after that 🤔